src/lib/tools.ts

```diff
@@ -151,12 +151,13 @@ export class LineHandler implements ToolHandler {
     y2: number,
     strict: boolean
   ): Promise<fabric.Line> => {
+    let [x, y] = [x2, y2];
     if (strict) {
       const rect = rectify(this.dirs, this.x, this.y, x2, y2);
-      x2 = rect[1];
-      y2 = rect[2];
+      x = rect[1];
```

Comment:
i believe a lint rule wants us to turn this to `[, x, y] = rect;` and i like that lint rule where is that

Reply:
I like that too actually

Reply:
I think you're looking for https://eslint.org/docs/rules/prefer-destructuring